### PR TITLE
Update of config in alfresco admin

### PR DESCRIPTION
**Pull request by svgta1**
- Allow relative path to doc server
- Allow onlyoffice config in json

### CODE REVIEW

**Comment by svgta1** [comment]
Updates for configuration in alfresco admin

File: repo/src/main/java/com/parashift/onlyoffice/Prepare.java [hunk 1]
```diff
@@ -187,6 +187,7 @@ public void execute(WebScriptRequest request, WebScriptResponse response) throws
                 editorConfigObject.put("lang", mesService.getLocale().toLanguageTag());
                 if (isReadOnly || preview || !canWrite) {
                     editorConfigObject.put("mode", "view");
+                    editorConfigObject.put("callbackUrl", callbackUrl);
```
Comment (LinneyS) [inline repo/src/main/java/com/parashift/onlyoffice/Prepare.java:190]: It is not required

**Comment by LinneyS** [comment]
`Allow relative path to doc server` - It's good idea
`Allow onlyoffice config in json` - for what ?

**Comment by LinneyS** [comment]
In #44 you write `Add json format for curl access control`
For what purposes do you need such access?

**Comment by svgta1** [comment]
> In #44 you write `Add json format for curl access control`
> For what purposes do you need such access?

An automated process can verify the config and then force value if needed without needed a humain to go in the config panel. And, with curl, a relative path is accepted.

**Pull request merged**